### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by ksobolew**
* Update the example to use the latest version
* Add a section on how to release a new version

### CODE REVIEW

**Comment by ksobolew** [comment]
It may or may not require updating the `maven-gpg-plugin`, too. I had some issues with the version in Airbase (1.4.0) and I actually did the release using 3.1.0, but the issue may have in fact been caused by lack of configured password to the GPG key.

File: README.md [hunk 1]
```diff
@@ -46,9 +46,85 @@ In `pom.xml`:
...
+```xml
+<settings>
+    <!-- ... -->
+    <profiles>
+        <profile>
+            <!-- this profile name is defined by Airbase -->
+            <id>oss-release</id>
+            <properties>
+                <gpg.skip>false</gpg.skip>
+                <gpg.passphrase><!-- your password to the private key here --></gpg.passphrase>
```
Comment (nevillelyh) [inline README.md:110]: I'd prefer not saving the passphrase in a file.

**Comment by nevillelyh** [comment]
You might need to configure GPG to prompt for passphrase. Mine looks like these:

```
> cat ~/.gnupg/gpg.conf
default-key ***** # in case there are multiple keys

❯ cat -p ~/.gnupg/gpg-agent.conf
default-cache-ttl 7200
max-cache-ttl 86400
pinentry-program /opt/homebrew/bin/pinentry-mac
```

Also need to `brew install pinentry-mac`, GUI app that prompts for passphrase.

**Comment by ksobolew** [comment]
> You might need to configure GPG to prompt for passphrase.

Yeah, the extremely confusing error message ("Invalid ioctl on a device") actually means that it can't read the password from the console, so that makes sense. I actually assumed that Maven won't be able to do it and I was under impression that `~/.m2/settings.xml` is the preferred way to store Maven credentials. But you're right that GPG password may be too sensitive for that.

**Comment by ksobolew** [comment]
I couldn't get prompting for a password to work, so I made a cop-out and I just pointed to the plugin documentation ;) I also kept setting the password in `~/.m2/settings.xml` as a less secure alternative, as this one is proven to work.

**Comment by ksobolew** [comment]
Ping

**Review by PutPixel — APPROVED** [review]

**Comment by ksobolew** [comment]
I'm merging this. We can improve the docs later, if needd.

**Pull request merged**